### PR TITLE
RSDK-7220 - Ml training wrappers

### PR DESCRIPTION
**Pull request by stuqdog**

### CODE REVIEW

File: lib/src/app/ml_training.dart
Comment (njooma) [inline lib/src/app/ml_training.dart:6]: `Ml` should be `ML`: https://dart.dev/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words

2 Letters should still be fully capitalized

File: lib/src/app/ml_training.dart
Comment (njooma) [inline lib/src/app/ml_training.dart:3]: This is the `MLTrainingClient` so this docstring is recursive lol. Should this be for the `ML Training Service`?

File: lib/src/app/ml_training.dart
Comment (njooma) [inline lib/src/app/ml_training.dart:30]: [nit] `registryItemId` does not have an explicit type

**Review by njooma — APPROVED** [review]
Content looks good. Just some minor naming changes please. Once fixed, feel free to merge without requesting another review

**Review by purplenicole730 — APPROVED** [review]
barring Naveed's comments

**Pull request merged**
